bin/dartdoc.dart

```diff
@@ -199,6 +200,8 @@ ArgParser _createArgsParser() {
   parser.addOption('hosted-url',
       help:
           'URL where the docs will be hosted (used to generate the sitemap).');
+  parser.addOption('example-path-prefix',
+      help: 'Prefix for @example paths. Default is project root.');
```

Comment:
Perhaps `help: 'Prefix for @example paths.\n(defaults to the project root)'`, to match the output for other options.